source/users.html.markdown

```diff
@@ -52,6 +52,7 @@ If you noticed something wrong with your entry here (or if you’d like us to re
 * [Imperial College](http://www.imperial.ac.uk/computing)
 * [InhouseIT](http://www.inhouseit.com)
 * [INRIA](http://inria.fr)
+* [KTH](http://cloud.cbh.kth.se)
```

Comment:
@pierrelefevre Should the link point to kth.se (main website)?

Reply:
Sure, if you prefer having the link point to the main homepage